client/src/index.tsx

```diff
@@ -38,12 +41,21 @@ const client = new ApolloClient({
     }
 });
 
+export const bugsnagClient = bugsnag({
+    apiKey: "f58d12ac7715ad9ffcfdd17fa3b93fc0",
```

Comment:
this apiKey isn't technically private but I should replace it with an environment variable